contracts/callpaths/ColdPath.sol

```diff
@@ -129,6 +130,9 @@ contract ColdPath is MarketSequencer, DepositDesk, ProtocolAccount {
         
         (int128 baseFlow, int128 quoteFlow) = initCurve(pool, price, initLiq);
         settleInitFlow(lockHolder_, base, baseFlow, quote, quoteFlow);
+
+        BeraCrocLpErc20 lpToken = new BeraCrocLpErc20(base, quote, poolIdx);
```

Comment:
I think we should use the `CREATE2` opcode to have deterministic addresses for counterfactual deployment.
Reference:
1. https://github.com/Uniswap/v2-core/blob/master/contracts/UniswapV2Factory.sol#L28-L32
2. https://blog.openzeppelin.com/getting-the-most-out-of-create2
3. https://book.getfoundry.sh/tutorials/create2-tutorial

Reply:
If we want to deploy Crocswap to the current testnet, we cannot use CREATE2 as it is not supported.

Reply:
That's a huge surprise.